libafl_bolts/src/llmp.rs

```diff
@@ -2242,15 +2268,38 @@ where
 
     /// Internal function, returns true when shuttdown is requested by a `SIGINT` signal
     #[inline]
-    #[cfg(unix)]
+    #[cfg(any(unix, all(windows, feature = "std")))]
     #[allow(clippy::unused_self)]
     fn is_shutting_down(&self) -> bool {
         unsafe { ptr::read_volatile(&LLMP_SIGHANDLER_STATE.shutting_down) }
     }
 
+    #[cfg(any(unix, all(windows, feature = "std")))]
+    fn _setup_signal_handler() -> Result<(), Error> {
+        #[cfg(all(unix, not(miri)))]
+        if let Err(_e) = unsafe { setup_signal_handler(&mut LLMP_SIGHANDLER_STATE) } {
```

Comment:
If it's used it doesn't need the underscore for `_` (it means "unused"), and in any case this can be merge with the lines below (they look identical?)

Ah I guess log::info is different - so they could have an extra `cfg`, up to you.

Reply:
Addressed